developer-docs-site/docs/tutorials/run-a-local-testnet.md

```diff
@@ -175,20 +173,23 @@ Completed generating configuration:
 Aptos is running, press ctrl-c to exit
 ```
 
-This output contains information required for starting the Aptos CLI tool:
+Use the [Aptos CLI tool](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/README.md) to interact with your local testnet. The above output contains information you will use for starting the [Aptos CLI tool](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/README.md):
+
 * `Aptos root key path`: The root key (also known as the mint or faucet key) controls the account that can mint tokens. Available in the docker compose folder under `aptos_root_key`.
 * `Waypoint`: A verifiable checkpoint of the blockchain (available in the docker compose folder under waypoint.txt)
 * `REST endpoint`: The endpoint for the REST service, e.g., `http://127.0.0.1:8080`.
-* `ChainId`: The chain id uniquely distinguishes this network from other blockchain networks.
+* `ChainId`: The chain ID uniquely distinguishes this network from other blockchain networks.
```

Comment:
I believe the `chainID` is no longer required while using the CLI tool `aptos`, I think? @gregnazario?